numpy/lib/arraysetops.py

```diff
@@ -332,14 +338,45 @@ def intersect1d(ar1, ar2, assume_unique=False):
     >>> from functools import reduce
     >>> reduce(np.intersect1d, ([1, 3, 4, 3], [3, 1, 2, 1], [6, 3, 4, 2]))
     array([3])
+    
+    To return the indices of the values common to the input arrays
+    along with the intersected values:
+    
+    >>> np.intersect1d([1,2,3,4], [2,1,4,6], return_indices=True)
+    (array([1,2,4]), array([0,1,3]), array([1,0,2]))
```

Comment:
Missing spaces around commas, especially in the output where what you have doesn't match what is printed.

I think this example would be more clear if made longer:
```python
>>> x = np.array([1,2,3,4])
>>> y = np.array([2,1,4,6])
>>> xy, x_ind, y_ind = np.intersect1d([1,2,3,4], [2,1,4,6], return_indices=True)
>>> xy
array([1, 2, 4])
>>> x_ind, y_ind
(array([0, 1, 3]), array([1, 0, 2]))
```